presentation/components/src/test/instance-filter-builder/PresentationInstanceFilterBuilder.test.tsx

```diff
@@ -63,6 +95,21 @@ describe("PresentationInstanceFilter", () => {
       return;
     }
     imodelMock.setup((x) => x.query(moq.It.isAnyString(), moq.It.isAny(), moq.It.isAny())).returns(() => generator());
+    imodelMock.setup((x) => x.key).returns(() => "test_imodel");
+    imodelMock.setup((x) => x.onClose).returns(() => onCloseEvent);
+    const metadataProvider = getIModelMetadataProvider(imodelMock.object);
+    sinon.stub(metadataProvider, "getECClassInfo").callsFake(async (id) => {
+      switch (id) {
+        case classInfo.id:
+          return new ECClassInfo(classInfo.id, classInfo.name, classInfo.label, new Set(), new Set());
+      }
+      return undefined;
```

Comment:
```suggestion
    sinon.stub(metadataProvider, "getECClassInfo").callsFake(async () => {
      return new ECClassInfo(classInfo.id, classInfo.name, classInfo.label, new Set(), new Set());
```

Looks like only this one class is used so there is no need to add `switch`.